package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "tp-todo-list-vue",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "vue": "^3.5.13"
```

Votre application ne fonctionne pas ! Impossible de la lancer sans la correction proposée.

C'est domage.

Veuillez à tester la soumission finale de vos projets (un `git clone` dans un dossier à part ne coûte rien).

```suggestion
    "vue": "^3.5.13",
    "vue-router": "^4.0.13"
```

---

Nous avions tester la soumission finale chacune de notre côté, et l'application marchait dans nos 2 cas. Je viens de refaire un clone à part et de même l'application fonctionne, donc je ne sais pas comment on aurait pu voir cette erreur.

---

Une explication possible à laquelle je peux pener c'est d'avoir installé vue-router *globallement* (avec `-g` lors de `npm install`). Une autre explication c'est si vous utilisez une vieille version de npm (avant 5) ou on devrait ajouter `--save` pour rendre l'installation permanente.

Je prends ton retour en compte dans la note final. La note nominale ne sera pas impacté par cette erreur car elle est liée uniquement au bonus.